.pre-commit-config.yaml

```diff
@@ -16,6 +16,8 @@ repos:
     rev: 21.12b0
```

Comment:
Upgrading black to `rev: 22.3.0` should negate the need to specify `click==8.0.4`.

Reply:
Upgrading to new black introduces more changes that are not relevant to this PR. That should be done in a separate branch.